less/core/block.less

```diff
@@ -1,11 +1,16 @@
 .block {
-  &.has-bg-image {
-    background-repeat: no-repeat;
-    background-size: cover;
-    background-position: center top;
+  position: relative;
+
+  &.has-bg-image > .background {
+    position: absolute;
+    top: 0;
+    height: 100%;
+    left: 0;
+    width: 100%;
   }
 
   &__inner {
+    position: relative;
```

Comment:
Could you expand on what you mean here?

Reply:
The page / menu / article have the `position: relative;` set up on the `&__header-inner` element. The block is the outlier on having it applied to `&__inner`. 

The relative is required for the header element to display but not the component area due to the relative positioning applied to the component.

Reply:
Is this comment just a general maintenance comment rather than related to this change?

Reply:
It's related to the background div change due to the addition of the `position: relative;`

Reply:
Upon reflection, it may actually be the case that the relativity on the block inner is correct but that the other implementations aren't quite correct. In essence the set up should be:

Parent = relative
Element = absolute
Sibling = relative

In this case, the `article__header-inner` elements `position: relative;` needs moving to `article__inner` and `position: relative;` needs adding to `page__inner`